posit-dev/positron
# Test older versions of pandas, torch

### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any GitHub issues that are related.

Instead of time travel, which might be erratic, specifically choose older major versions of packages to run against.

Versions of packages for which we have specific inspectors:

```
pandas: has 1.X, 2.X
torch: has 1.X, 2.X
polars: still on 0.X, just test latest
numpy: has been 1.X since 2006, soon to release 2.X
```

related to #2302 #2597 

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

I've changed the name of the requirements file and specifically pinned older versions of packages. These are both the last release of 1.X versions.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues.

You should be able to test this locally with `yarn testMinimumPythonReqs`. Note that you must use Python 3.9 or lower, since torch 1.X cannot be installed on later versions.
